Source/BSGConnectivity.m

```diff
@@ -95,8 +90,10 @@ - (void)dealloc {
 - (void)startWatchingConnectivity {
     SCNetworkReachabilityContext context = {
         .version = 0,
-        .info = (void *)CFBridgingRetain(self.callbackBlock),
-        .release = CFRelease};
+        .info = (__bridge void *)self.callbackBlock,
```

Comment:
This change will prevent the static analyser from detecting this area as a potential memory leak. As we're passing in the retain AND release function to `SCNetworkReachabilitySetCallback` it will be responsible for both retaining and releasing.

Otherwise the static analyser will detect that we are calling `CFBridgingRetain` ourselves and flag as a potential leak as it never sees it released.